components/survey/element.css

```diff
@@ -9,6 +9,7 @@
 
   padding: 1rem;
   padding-left: 5rem;
+  margin-top: 1.6rem;
```

Comment:
Nit: How do we get to 1.6rem, why not 1.5rem?

Reply:
1.5rem is the top margin of the title, but since it's line height isn't 1, there's internal padding between the top of the character and the bounding box, so 1.5rem makes the banner look a bit lopsided visually: 1.6rem looked best to my eye, balancing the padding with aligning with the sidebar filter to the left and "In this article" heading to the right